filter/src/main/java/com/yalantis/filter/widget/Filter.kt

```diff
@@ -349,15 +324,16 @@ class Filter<T : FilterModel> : FrameLayout, FilterItemListener, CollapseListene
             putBoolean(STATE_COLLAPSED, isCollapsed!!)
             val selected = mSelectedItems
             val removed = mRemovedItems
-            if (selected is Serializable) {
+            if (true) {
```

Comment:
Why not remove this line and 330?

Reply:
Android Studio tell me it is always true

Reply:
```
if (true) {
    putSerializable(STATE_SELECTED, selected)
}
```
... is the same as ...

```
putSerializable(STATE_SELECTED, selected)
```

just remove the **if condition**